sdk/core/core/inc/az_http_response_action.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#ifndef AZ_HTTP_RESPONSE_ACTION_H
+#define AZ_HTTP_RESPONSE_ACTION_H
+
+#include <az_span_emitter.h>
+#include <az_pair.h>
+#include <az_result.h>
+#include <az_span.h>
+
+#include <_az_cfg_prefix.h>
+
+typedef struct {
+  az_span_action header_name_part_action;
+  az_span_action header_value_part_action;
+  az_span_action body_part_action;
+} az_http_response_action;
+
+AZ_ACTION_TYPE(az_http_response_emitter, az_http_response_action)
```

Comment:
see I've already forgotten what this macro does. I would _really_ prefer just writing out the type.